### PR TITLE
Redux migration/augment ideas reducer

### PR DESCRIPTION
**Pull request by vanderhoop**
hey @caporta @crq @samdec11 @kimberlysuazo, thought i'd open a tiny PR for augmenting our ideas action creator and reducer so that it can handle the initial setting of pre-existing ideas when joining an in-progress retro. I'm new to redux, and while I think I'm on point, I appreciate any feedback. thanks!

### CODE REVIEW

File: web/static/js/reducers/idea.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 const idea = (state = [], action) => {
   switch (action.type) {
+    case "SET_IDEAS":
```
Comment (kimberlysuazo) [inline web/static/js/reducers/idea.js]: @vanderhoop If the idea is to fetch all of the ideas, I think this looks good. Would this be the same as ```return action.ideas``` ?

File: web/static/js/reducers/idea.js [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
+      return [...action.ideas]
```
Comment (kimberlysuazo) [inline web/static/js/reducers/idea.js]: @vanderhoop If the idea is to fetch all of the ideas, I think this looks good. Would this be the same as ```return action.ideas``` ?

Comment (caporta) [inline web/static/js/reducers/idea.js]: @vanderhoop I like it! and 👍 to @kimberlysuazo's idea here. returning `action.ideas` outright seems the most straightforward approach, and will preserve immutability as it's a brand new array

Comment (vanderhoop) [inline web/static/js/reducers/idea.js]: glaring oversight on my part. thanks for the spot, @kimberlysuazo!

**Pull request merged**